tensorboard/plugins/core/core_plugin.py

```diff
@@ -23,7 +23,6 @@
 import gzip
 import math
 import mimetypes
-import os
```

Comment:
Personally I’d put this in a separate PR as well, but I could be
convinced to let this one slide. :-)

Reply:
I'll remove it for cleanliness